condaci.py

```diff
@@ -615,7 +615,8 @@ def binstar_upload_and_purge(mc, key, user, channel, filepath):
 
 is_pr_from_travis = lambda: os.environ['TRAVIS_PULL_REQUEST'] != 'false'
 is_pr_from_appveyor = lambda: 'APPVEYOR_PULL_REQUEST_NUMBER' in os.environ
-is_pr_from_jenkins = lambda: 'ghprbSourceBranch' in os.environ
+# TODO: Remove when ghprb is fixed
+is_pr_from_jenkins = lambda: os.environ['JOB_NAME'].split('/')[0][-3:] == '-pr'
```

Comment:
Added a note to `menpo-admin`: https://github.com/menpo/menpo-admin/commit/073d12ed9dd197f394c88ad80c30510e32b25bc2